content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -553,3 +553,172 @@ $ curl \
   }
 }
 ```
+
+
+## Associate an SSH key to a workspace
+
+This endpoint associates an SSH key to a workspace.
+
+| Method | Path           |
+| :----- | :------------- |
+| PATCH | /workspaces/:workspace_id/relationships/ssh-key |
+
+### Parameters
+
+* `:workspace_id` (`string: <required>`) - Specifies the workspace ID to associate the SSH key to.
+* `id` (`string: <required>`) - Specifies the SSH key ID to associate.
```

Comment:
Could use a quick mention of which endpoint I should use to obtain an SSH key ID.

Reply:
...Actually, is it an undocumented endpoint? I'm not finding anything about SSH keys in a quick search.

Reply:
@nfagerlund yeah, it is undocumented. Shall I document it as part of this PR or handle it separately?

Reply:
@nfagerlund looks like the API is really required here I guess, as there's no way in the UI to gather this information, unless we told them to inspect the source or added the ID to the UI on the manage page.

Reply:
In that case, I think add the documentation in this PR. We'll need a new page and a new item in the nav sidebar. IDK if you've done the sidebar business before, but the file is at content/source/layouts/enterprise2.erb. The only really squirrelly part is the `sidebar_current` business, but you can probably figure it out from the existing examples.

Reply:
Sounds good. Thanks for the pointers.